src/main/java/com/teamabnormals/abnormals_core/core/util/ACHooks.java

```diff
@@ -0,0 +1,35 @@
+package com.teamabnormals.abnormals_core.core.util;
+
+import com.teamabnormals.abnormals_core.client.screen.SlabfishHatScreen;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screen.CustomizeSkinScreen;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.widget.Widget;
+import net.minecraft.client.gui.widget.button.Button;
+import net.minecraft.util.text.Color;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public final class ACHooks {
+	private static final Method ADD_BUTTON_METHOD = ObfuscationReflectionHelper.findMethod(Screen.class, "func_230480_a_", Widget.class);
+
+	/**
+	 * Called via ASM - customization_screen.js Transformer
+	 */
+	public static void addSlabfishButton(CustomizeSkinScreen screen, int i) {
```

Comment:
Would be good to have strings like `"abnormals_core.screen.slabfish_settings"` be a constant field in this class.

Reply:
Decided to move this into `SlabfishHatScreen` since that's where the actual screen stuff is handled.

Reply:
I still think those strings should be constant though.